folio-integration-kafka/src/main/java/org/folio/integration/kafka/KafkaTopicConfiguration.java

```diff
@@ -17,6 +18,11 @@ public class KafkaTopicConfiguration {
 
   @PostConstruct
   public void createTopics() {
+    if (isEmpty(folioKafkaProperties.getTopics())) {
+      log.debug("No topics to create");
+      return;
+    }
+
     for (var topic : folioKafkaProperties.getTopics()) {
```

Comment:
alternatively to avoid NPE you can [`emptyIfNull()](https://commons.apache.org/proper/commons-collections/javadocs/api-4.2/org/apache/commons/collections4/ListUtils.html#emptyIfNull-java.util.List-)` method from `ListUtils`